src/hash/mod.rs

```diff
@@ -1,9 +1,9 @@
 //! Cryptographic hash functions used by the Miden VM and the Miden rollup.
 
-use super::{Felt, FieldElement, StarkField, ONE, ZERO};
+use super::{CubeExtension, Felt, FieldElement, StarkField, ONE, ZERO};
 
 pub mod blake;
-pub mod rpo;
+pub mod rescue_prime;
```

Comment:
To reduce the amount of changes I'd probably re-export RPO and RPX at the same level. So, for example, we could replace this line with something like:
```Rust
mod rescue;

pub mod rpo {
    pub use super::rescue::{Rpo256, RpoDigest};
}

pub mod rpx {
    pub use super::rescue::{Rpx256, RpxDigest};
}
```